staff/serena-mejias/gamma-template/about/about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <style>
+      @import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800");
+    </style>
+    <link rel="stylesheet" href="reset.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
+      integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="head">
+      <h3 class="head__title">Gamma</h3>
+      <i class="fas fa-bars head__icon"></i>
+      <ul class="head__menu">
+        <li class="head__menu__item">Home</li>
```

Comment:
You shouldn't have to elements inside the same block, block__elem__elem. You should have only block__elem. If you  have a should element you need to convert the first element to a block.

Reply:
corrected